cylc/flow/network/schema.py

```diff
@@ -1547,7 +1547,7 @@ class Meta:
 
     class Arguments:
         workflows = graphene.List(WorkflowID, required=True)
-        task_job = String(required=True)
+        job_id = String(required=True)
```

Comment:
Note change in GraphQL interface, I assume this is ok since this is a minor rather than maintenance release?

Reply:
Unfortunately we do not have ANY strategy for handling GraphQL schema changes at the moment.

Because the UI Server uses a single GraphQL endpoint for all workflows the schemas must be consistent between all running workflows (or the resulting errors must be handled).

Reply:
Removing the schema change, only keeping the internal python variables rename...